src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/repository/VideoLinkBookingRepositoryTest.kt

```diff
@@ -307,11 +308,11 @@ class VideoLinkBookingRepositoryTest(
      * CourtService#updateVideoLinkBooking depends on this behaviour to add those ids to an Application Insights custom
      * event.
      */
-    assertThat(booking.appointments.values).allSatisfy { assertThat(it.id).isNull() }
```

Comment:
There is a new overloaded method allSatisfy so I had to specify which one to call